src/en/space-station-14/round-flow/antagonists/Xenoborgs.md

```diff
@@ -0,0 +1,194 @@
+- You must protect your own existance at all costs
+- You must protect the existence of all xenoborgs
+- You must create more xenoborgs
+- Get your xenoborgs to deliver you steel to make xenoborg shells
+- Get your xenoborgs to deliver you brains of sentient beings to fill the xenoborg shells to complete them
+
+They are also able to produce better versions of some xenoborg specific modules in exchange for glass, to upgrade the xenoborgs.
+
+They see with cameras and act like some sort of AI of the mothership.
+
+The mothership core has the same access as all the xenoborgs
+
+Besides the normal xenoborg radio, the mothership has a special radio frequency that only they can speak on but all xenoborgs can hear, this is so they can send more direct instructions and to coordinate the xenoborgs better.
+
+<!-- Give a description of what game mechanics you would like to add or change. This should be a general overview, with enough details on critical design points that someone can directly implement the feature from this design document. Exact numbers for game balance however are not necessary, as these can be adjusted later either during development or after it has been implemented, but mention *what* will have to be balanced and what needs to be considered when doing so. -->
+
+## Round resolution
+
+The objective of the Xenoborgs is to borg a certain percentage of the crew (50% is a good number I believe, but this can be decided later). Once this objective is complete, the evacuation shuttle is called automatically (can't be recalled) but of course the Xenoborgs can still try to borg more people before the round actually ends.
+
+If all the xenoborgs die or are in a crit state, then the mothership core explodes (big explosion, destroys the mothership)
+If the mothership core is destroyed (it explodes, but not a big explosion) and all the xenoborgs explode (not a big explosion)
+
+Major xenoborg win is to complete the objective to borg X% of the crew.
+Minor xenoborg win is to survive with at least a certain numbers of xenoborgs
+Minor crew win is to escape and leave less than a certain ammount of xenoborgs alive
+Major crew win is to destroy the mothership (either killing all the xenoborgs or destroying the core)
+
+The end summary will also tell how much of the crew were borged, and how many xenoborgs were there in the end.
+
+In the Future the Xenoborgs could have custom objectives such as create another core in the station or mess with the Station AI somehow
+
+## Game Design Rationale
+
+A army of evil robots saying "the flesh is weak, join us" while bashing a poor crewmate to death and dragging them away is certainly chaotic and silly.
+
+It is always enjoyable to play as murderbone antag, so playing as xenoborg is fun. since you are a murderous cyborg you get to roleplay as such and be a deranged borg obsessed with helping the "mother" or saying all organics are weak and the steel is stronger, only to die to a salvager with a lecter.
```

Comment:
```suggestion
It is always enjoyable to play as murderbone antag, so playing as xenoborg is fun. Since you are a murderous cyborg you get to roleplay as such and be a deranged borg obsessed with helping the "mother" or saying all organics are weak and the steel is stronger, only to die to a salvager with a Lecter.
```